openretina/hoefling_2024/nnfabrik_model_loading.py

```diff
@@ -95,13 +97,16 @@ def get_model(
     )
 
     if state_dict is not None:
+        ignore_missing = model_config.get("transfer", False)
+        if not strict:
+            ignore_missing = True
```

Comment:
Could write that shorter
```suggestion
        ignore_missing = model_config.get("transfer", False) or not strict
```